thumbor/config.py

```diff
@@ -248,6 +248,13 @@
     "Imaging",
 )
 
+Config.define(
+    "PRESERVE_EXIF_COPYRIGHT_INFO",
+    False,
+    "Preserves exif copyright information in generated images.",
```

Comment:
Nit:
```suggestion
    "Preserves Exif copyright information in generated images.",
```
According to https://en.wikipedia.org/wiki/Exif:
> [...] officially Exif, according to JEIDA/JEITA/CIPA specifications [...]